webapp/sources/rudder/rudder-web/src/test/scala/bootstrap/liftweb/checks/migration/TestMigrateChangeValidationEnforceSchema.scala

```diff
@@ -63,19 +63,6 @@ class TestMigrateChangeValidationEnforceSchema extends DBCommon with IOChecker {
 
   override def transactor: Transactor[cats.effect.IO] = doobie.xaio
 
-  override def cleanDb(): Unit = {
-    doobie.transactRunEither(
-      sql"""
-        DROP TABLE IF EXISTS ${tempWorkflowTable};
-        DROP TABLE IF EXISTS ${tempCRTable};
-        DROP SEQUENCE IF EXISTS ChangeRequestId_temp;
-      """.update.run.transact(_)
-    ) match {
-      case Right(_) => ()
-      case Left(ex) => throw ex
-    }
-  }
-
```

Comment:
Since the test schema is dropped anyway, every table/sequence/etc. created during tests is guaranteed to also be dropped